website/docs/guides/1.13.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.13.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1130-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.13.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.13.0: Upgrade and Information Guide
```

Comment:
this PR https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1640 has a breaking change which i think we should list here

Reply:
@oarbusi Did you use the correct tag when generating the changelog? You need to update the tag to 1.13.1-pre before running the github changelog binary. I am asking because the github changelog should have picked automatically the breaking change

Reply:
@andreaangiolillo #1640 was merged after v1.13.0 as far as I understand, so not included in that release which this upgrade guide refers to.

Reply:
I see it is present here: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1656 (1.13.1 change log), not sure if it was manually moved away from Breaking Changes section

Reply:
Good point. I thought this was the changelog 🤦 . Moving this conversation to the other PR. Thanks!